elastica/dissipation.py

```diff
@@ -151,3 +154,115 @@ def dampen_rates(self, rod: SystemType, time: float):
         rod.omega_collection[:] = rod.omega_collection * np.power(
             self.rotational_exponential_damping_coefficient, rod.dilatation
         )
+
+
+class FilterDamper(DamperBase):
+    """
+    Filter damper class. This class corresponds qualitatively to a low-pass filter
+    applied on the translational and rotational velocities, filtering out the high
+    frequency (noise) modes, while having no effect on the low frequency smooth modes.
+
+    Examples
+    --------
+    How to set filter damper for rod:
+
+    >>> simulator.dampen(rod).using(
+    ...     FilterDamper,
+    ...     filter_order=3,   # order of the filter
+    ... )
+
+    Notes
+    -----
+    The extent of filtering can be controlled by the `filter_order`. Small
+    integer values (1, 2, etc.) result in aggressive filtering, and can lead to
+    the "physics" being filtered out. While high values (9, 10, etc.) imply
+    minimal filtering, and thus negligible effect on the velocities.
+    Values in the range of 3-7 are usually recommended.
+
+    For details regarding the numerics behind the filtering, refer to:
+
+    Jeanmart, H., & Winckelmans, G. (2007). Investigation of eddy-viscosity
+    models modified using discrete filters: a simplified “regularized variational
+    multiscale model” and an “enhanced field model”. Physics of fluids, 19(5), 055110.
```

Comment:
Can we add thesis as reference.

Reply:
The thesis takes the idea from this paper, so is it needed since we already cited the original idea people?

Reply:
Thesis can be the other reference, I think there they present the idea nicely.

Reply:
```suggestion
    multiscale model” and an “enhanced field model”. Physics of fluids, 19(5), 055110.
    .. [2] Lorieul, G. (2018). Development and validation of a 2D Vortex Particle-Mesh
     method for incompressible multiphase flows 
     (Doctoral dissertation, Université Catholique de Louvain).
```